packages/connect-wallet-modal/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@reef-knot/connect-wallet-modal",
+  "version": "0.0.1",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/reef-knot",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/reef-knot.git",
+    "directory": "packages/connect-wallet-modal"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/reef-knot/issues"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w"
+  },
+  "peerDependencies": {
+    "@lido-sdk/constants": ">=1.6",
+    "@lido-sdk/react": ">=1.18",
+    "@lido-sdk/web3-react": ">=1.25",
+    "@lidofinance/lido-ui": ">=2.9",
+    "react": ">=16",
+    "react-dom": ">=16",
+    "react-is": ">=16",
+    "styled-components": "5",
+    "ua-parser-js": ">=1.0.2"
```

Comment:
This also looks a bit uncommon for general consumer if we expect this to be used outside of Lido

Reply:
The current v0 version is not prepared for an external consumer